pkg/network/kuryr.go

```diff
@@ -157,6 +157,15 @@ func validateKuryr(conf *operv1.NetworkSpec) []error {
 		octaviaServiceNet = &octaviaServiceNetObj
 	}
 
+	if kc != nil && kc.EnablePortPoolsPrepopulation && kc.PoolBatchPorts != nil && *kc.PoolBatchPorts > 0 {
+		if kc.PoolMinPorts > 0 && *kc.PoolBatchPorts < kc.PoolMinPorts {
```

Comment:
we should also warn about batchports to at least be 1, right?

Reply:
Right!